swagger.yaml

```diff
@@ -2033,8 +2033,8 @@ paths:
     get:
       operationId: listCollections
       summary: List Collections
-      description: Returns a representation of `Collection`s in a `Workspace` visible to
-        the authenticated `User`
+      description: Returns a representation of `Collections`s in a `Workspace` visible
```

Comment:
I'm guessing this shouldn't be `'Collections's` (note the double "s").

Reply:
PR to fix in webapp: https://github.com/mode/webapp/pull/15558